src/pages/loading-waiting-states/progress-bar-indeterminate/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { AppBar, Toolbar, Typography, Hidden, IconButton } from '@material-ui/core';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import MenuIcon from '@material-ui/icons/Menu';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { EmptyState } from '@pxblue/react-components';
+import { BluetoothSearching } from '@material-ui/icons';
+import LinearProgress from '@material-ui/core/LinearProgress';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        height: `calc(100vh - ${theme.spacing(8)}px)`,
+    },
+    appbarRoot: {
+        padding: 0,
+    },
```

Comment:
I don't think this class is necessary...the AppBar should already have no padding by default.